components/embl-group-page/embl-group-page.hbs

```diff
@@ -42,7 +42,7 @@
 
     <section class="embl-grid">
       <nav class="vf-breadcrumbs embl-breadcrumbs-lookup" aria-label="Breadcrumb" data-embl-js-breadcrumbs-lookup>
-        <div class="vf-text embl-breadcrumbs-lookup--loading-text"></div>
+        <div class="vf-text embl-breadcrumbs-lookup--ghosting"></div>
```

Comment:
if this is now no longer text - do we need `vf-text` and if it is 'text' as it's loading in text should it be a `p` rather than a `div`?

Reply:
Probably don't need the text. I'll double check

Reply:
Ah, actually just failed to fully update the group-page.hbs example; should have been `  <div class="vf-list vf-list--inline | vf-breadcrumbs__list | embl-breadcrumbs-lookup--ghosting"></div>`

Sorted now.